### ISSUE TITLE
Filter-bar : Screen reader should convey ‘Checked’ information for the checked checkbox and ‘Unchecked’ information for the unchecked checkbox both the information should be avoided[Component Name]: [Bug Description]

### ISSUE THREAD

**Issue by sanjayC5373903** [SAP/ui5-webcomponents-react#5982]
### Describe the bug

In entire application while navigating around the checkboxes within ‘Filters’ dialog, both ‘Checked and unchecked’ information is being read. Screen reader is not working correctly.

![Picture2](https://github.com/SAP/ui5-webcomponents-react/assets/170628934/e08126dc-c73e-4a0c-908e-1063e5dd339a)

<S.FilterBar
      filterBarExpanded
      showGoOnFB
      showClearOnFB
      onRestore={onFilterBarRestore}
      useToolbar={useToolbar}
      showFilterConfiguration={showFilterConfiguration}
      onGo={onGo}
      showRestoreButton={showRestoreButton}
      onClear={onClear}
      data-testid="FilterBarComp"
      filterContainerWidth="17rem"
      className="sapUiSizeCompact"
      onFiltersDialogSave={e => handleOnFiltersDialogSave(e, onFilterConfigDialogSave)}
      onFiltersDialogCancel={onFiltersDialogCancel}
      onFiltersDialogOpen={e => handleOnFiltersDialogOpen(e, onFilterConfigDialogOpen)}
      onFiltersDialogSelectionChange={onFiltersDialogSelectionChange}
      variants={     
        <VariantManagementField
          variantList={variantList}
          updateVariantId={updateVariantId}
          currentVariant={currentVariant}
          isVariantDirty={isVariantDirty}
          onSaveManageView={onSaveManageView}
          handleVariantCreate={handleVariantCreate}
          handleVariantEdit={handleVariantEdit}
          triggerDataFetch={triggerDataFetch}
          variantId={variantId}
          triggerVariantDetailFetch={triggerVariantDetailFetch}
        />
      }
    >
      {filterFieldsList.map(
        ({
          key,
          fieldKey,
          label,
          groupName,
          labelTooltip,
          required,
          visibleInFilterBar,
          enabled = true,
          considerGroupName,
          ...filter
        }) => (
          <FilterGroupItem
            key={key}
            label={label}
            groupName={groupName}
            labelTooltip={labelTooltip}
            required={required}
            considerGroupName={considerGroupName}
            data-testid="FilterGroupItemComp"
            visibleInFilterBar={enabled}
          >
            <FilterField
              label={label}
              key={key}
              fieldKey={fieldKey}
              {...filter}
              required={required}
              inputDataset={filterDataset}
              customOnChangeHandler={onChange}
              onPageFilterBarGoOnEnter={onPageFilterBarGoOnEnter}
              onGo={onGo}
            />
          </FilterGroupItem>
        ),
      )}

    </S.FilterBar>
    
    ------------------------------------------------------------------------------
    import React, { forwardRef } from 'react';

const FilterField = forwardRef(({ as: FilterFieldComponent, ...props }, ref) => {
  return <FilterFieldComponent {...props} />;
});

export default FilterField;

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

Screen reader should convey ‘Checked’ information for the checked checkbox and ‘Unchecked’ information for the unchecked checkbox both the information should be avoided.

### UI5 Web Components for React Version

0.27.5

### UI5 Web Components Version

1.6.0

### Browser

Chrome

### Operating System

Windows11

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @sanjayC5373903 

v0.27.5 was still in beta state and is over 2 years old. Since I can't reproduce the issue and the FilterBar has changed a lot since then please retest the issue with the latest version of `@ui5/webcomponents-react`.
